### PR TITLE
[feat] Team modal window

### PR DESCRIPTION
**Pull request by rudenko-vl**
- Add html markup for the modal window
- Add modal window styles
- Add styles for breakpoints
- I add additional links to the refs.js file
- Add social media icons to the sprite.svg
- Created a file team-modal-js to open and close a modal window
- Add import of file team-modal.js to index.js

### CODE REVIEW

File: src/js/modules/components/team-modal.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+teamModalBtnClose.addEventListener('click', closeTeamModal);
+window.addEventListener('click', closeTeamModal);
+window.addEventListener('keydown', closeTeamModal);
+
+function openTeamModal() {
+  backdrop.classList.remove('is-hidden');
+}
+
+function closeTeamModal(ev) {
+  if (ev.target === backdrop || ev.keyCode === 27 || ev.target === teamModalBtnClose) {
```
Comment (Androborn) [inline src/js/modules/components/team-modal.js:14]: What do these checks do?

File: src/images/sprite.svg [hunk 2]
```diff
@@ -1 +1,145 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+</symbol>
+</svg>
```
Comment (Androborn) [inline src/images/sprite.svg]: Please optimize and minimize sprite.

**Review by Androborn — CHANGES_REQUESTED** [review]

File: src/images/sprite.svg (hunk 2 above)
Comment (rudenko-vl) [reply src/images/sprite.svg]: Done

File: src/js/modules/components/team-modal.js (hunk 1 above)
Comment (rudenko-vl) [reply src/js/modules/components/team-modal.js:14]: These conditions check that the window is closed by clicking on a button, on a backdrop or pressing the Esc button.

**Review by Androborn — APPROVED** [review]

**Pull request merged**